workflow-service/src/main/java/com/redhat/parodos/workflow/execution/continuation/WorkFlowContinuationService.java

```diff
@@ -99,8 +102,12 @@ public void workFlowRunAfterStartup() {
 							throw new RuntimeException(e);
 						}
 					});
+					Map<String, String> workFlowArguments;
+					workFlowArguments = WorkFlowDTOUtil.readStringAsObject(workFlowExecution.getArguments(),
```

Comment:
I would add this to the test, there is already one, so added the check will not hurt